### ISSUE TITLE
WAVE extension contrast issues

### ISSUE THREAD

**Issue by lcaple** [uweb/uw_wp_theme#36]
From Terrill: 

> I'm seeing new contrast issues in the new theme, as measured by the [WAVE browser extension](https://wave.webaim.org/extension/) on both the [Admissions](https://www.washington.edu/admissions/) and [CISO](https://ciso.uw.edu/) websites. The Admissions site has 5 errors, whereas the CISO site has 11. The 5 errors on the Admissions site are shared on both sites. Keep in mind that some of these errors may be the result of a CSS background-color either having too little contrast or not being defined at all, even if the background-color is hidden by a background image. This is problematic when the image fails to load, or is slow to load, and users need to be able to read the text.

**Comment by lcaple** [comment]
Resolved in [1.3.0](https://github.com/uweb/uw_wp_theme/releases/tag/1.3.0)
